### PR TITLE
Ganglia 3.7.1"

### PR DESCRIPTION
**Pull request by Philippe-Caparroy**

### CODE REVIEW

**Comment by DomT4** [comment]
Please run `brew audit --strict` on this locally. There's a decent amount that will need fixing, from double quotes to sourceforge url to checksum type and so on.

**Comment by Philippe-Caparroy** [comment]
Ok, it's done.

**Comment by DomT4** [comment]
Missed a few:

```
==> audit problems
ganglia371:
 * A `test do` test block should be added
 * Stable: SHA1 checksums are deprecated, please use SHA256
 * Stable: Don't use /download in SourceForge urls (url is http://sourceforge.net/projects/ganglia/files/ganglia%20monitoring%20core/3.7.1/ganglia-3.7.1.tar.gz/download).
 * Stable: Use http://downloads.sourceforge.net to get geolocation (url is http://sourceforge.net/projects/ganglia/files/ganglia%20monitoring%20core/3.7.1/ganglia-3.7.1.tar.gz/download).
 * Use `system "make", "install"` instead of `system "make install"` 
```

**Comment by Philippe-Caparroy** [comment]
Should be fine now.

**Comment by DomT4** [comment]
The test failed:

```
==> brew test --verbose Homebrew/versions/ganglia371			 FAILED
Testing ganglia371
==> /usr/local/Cellar/ganglia371/3.7.1/bin/ganglia371 --version
Failed to execute: /usr/local/Cellar/ganglia371/3.7.1/bin/ganglia371
Error: ganglia371: failed
Failed executing: /usr/local/Cellar/ganglia371/3.7.1/bin/ganglia371 --version
```

You need to open another PR for `Solr` unless it's necessary for the two to be bunched together, i.e. one depends on the other, etc. Otherwise it gets difficult to know where each formula came from for future submitters.

**Comment by DomT4** [comment]
You need to squash your commits here. It's presuming that `solr51` still exists because it previously did.

File: ganglia371.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+class Ganglia371 < Formula
+  homepage "http://ganglia.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/ganglia/ganglia%20monitoring%20core/3.7.1/ganglia-3.7.1.tar.gz"
+  sha256 "e735a6218986a0ff77c737e5888426b103196c12dc2d679494ca9a4269ca69a3"
+  conflicts_with "coreutils", :because => "both install `gstat` binaries"
```
Comment (DomT4) [inline ganglia371.rb]: Put this under the dependencies.

File: ganglia371.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
+class Ganglia371 < Formula
+  homepage "http://ganglia.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/ganglia/ganglia%20monitoring%20core/3.7.1/ganglia-3.7.1.tar.gz"
```
Comment (DomT4) [inline ganglia371.rb]: Out of interest, why was this submitted here and not as an update to the existing formula in Homebrew/homebrew? Is it pre-release or drastically incompatible or something?

**Comment by DomT4** [comment]
Closed per https://github.com/Homebrew/homebrew-versions/pull/832#issuecomment-103081479. I'll reopen if it becomes necessary for whatever reason later.

**Pull request closed without merging**